lib/middleware/errors.js

```diff
@@ -47,8 +44,33 @@ function onUncaughtException(error) {
   setImmediate(process.exit.bind(null, 66), 1000);// eslint-disable-line no-undef
 }
 
+function renderView(err, req, res, next) {
+  const statusCode = err.status || 500;
+  const errorMessages = getErrorMessage(statusCode);
+
+  const viewData = {
```

Comment:
Don't think we need an extra object. We can just add this to the `const data` one.
Also, I think there are some more vars required by the footer:
```
FOOTER_THEME
FOOTER_TYPE
FOOTER_PADDING_TOP
FOOTER_HELP_LINK
```